fab/inventory/icds

```diff
@@ -72,28 +109,35 @@ kafka0
 
 [celery:children]
 celery0
+celery1
 
 [pillowtop:children]
 pil0
```

Comment:
We will need to add `pil1` here after we update HQ to support multiple pillow machines